windows/src/developer/TIKE/compile/MergeKeyboardInfo.pas

```diff
@@ -289,21 +291,57 @@ function TMergeKeyboardInfo.LoadKMPFile: Boolean;
       end;
 
       FKMXTempFile := TTempFileManager.Get('.kmx');
+      FJSTempFile := TTempFileManager.Get('.js');
```

Comment:
We don't need this temp file because we are not processing js file contents in the same way as kmx file contents?